test/cli/test_gen.py

```diff
@@ -26,8 +26,8 @@ def test_missing_args(self):
         self.assertEqual(1, result.exit_code)
 
     def test_main_with_illegal_size_option(self):
-        result = self.invoke_cli(['-s', '120,abc', 'input.nc'])
-        self.assertEqual(1, result.exit_code)
+        exit_code = self.invoke_cli(['-s', '120,abc', 'input.nc'])
```

Comment:
I was not aware that `result`*has an* `exit_code`. Therefore "result" was fine, SORRY!